pkg/controller/operator.go

```diff
@@ -191,5 +191,5 @@ func (s *Support) Run(ctx context.Context, controller *controllercmd.ControllerC
 	go uploader.Run(ctx)
 
 	<-ctx.Done()
-	return fmt.Errorf("stopped")
+	return nil
```

Comment:
Could I please ask you to add before this line log something like "stopped". I think it would be slight indication the logs in case we would investigate some crash or refresh (we are seeing that it is triggered, but this would help to see observer really finished).

Reply:
Sure. I added it as a warning message to the log.